examples/server_hardware_type.go

```diff
@@ -10,8 +10,8 @@ import (
 func main() {
 	var (
 		clientOV                  *ov.OVClient
-		server_hardware_type_name = "SY 480 Gen9 1"
-		uri                       = "/rest/server-hardware-types/D3DCBFC4-E421-4B47-801D-B46E28D99482"
+		server_hardware_type_name = "SY 480 Gen9 2"
```

Comment:
we don't need these changes i suppose

Reply:
Its just an example file.  And an indication of the values i've used to test the api1000 support.